src/main/java/yoyo/command/Command.java

```diff
@@ -0,0 +1,54 @@
+package yoyo.command;
+
+import java.util.Arrays;
+
+import yoyo.core.Storage;
+import yoyo.core.Ui;
+import yoyo.exception.YoyoException;
+import yoyo.task.TaskList;
+
+
+public abstract class Command {
+    protected String[] inputTokens;
+
+    public Command(String[] inputTokens) {
+        this.inputTokens = inputTokens;
+    }
+
+    /**
+     * Checks remaining user input for incomplete commands.
+     *
+     * @param inputTokens String array from user input.
+     * @throws YoyoException.YoyoIncompleteCommandException Thrown if command is incomplete.
+     */
+    public static void checkCompleteCommand(String[] inputTokens)
+            throws YoyoException.YoyoIncompleteCommandException {
+        if (inputTokens.length < 2 || inputTokens[1].trim().length() == 0) {
+            throw new YoyoException.YoyoIncompleteCommandException(
+                    "You have not entered enough information for your command.");
+        }
+    }
+
+    public boolean shouldContinueProgram() {
+        return true;
+    }
+
+    /**
+     * Executes actions depending on command type.
+     *
+     * @param tasks Tasks currently in the Yoyo program.
+     * @param storage Storage instance of the Yoyo program.
+     * @param ui Ui instance of Yoyo program.
+     * @throws YoyoException
```

Comment:
Perhaps to be consistent with your previous header comment, more information on when this exception would be thrown could be given to assist the reader in understanding the method?